### DISCUSSION TITLE
.p4config file showing on opengrok UI with latest 1.10

### DISCUSSION THREAD

**Discussion by jkjha** [oracle/opengrok#4273] [Q&A]
I recently upgraded to the latest version of OpenGrok (v1.10) for bug fixes such as those discussed in https://github.com/oracle/opengrok/discussions/4252. 

However, I noticed that the UI is displaying a .p4config file containing sensitive information such as client and server names. While I cannot confirm whether this is related to the upgrade, as I cannot replicate the issue with a smaller dataset on another server, it is visible on my primary server. 

Do any one have any insights into what might be causing this?

Having .p4config file under /opengrok/src/<project_name>/.p4config is required to sync the projects and run different commands there. Hence I cann't delete that.

**Comment by vladak** [comment]
I think you are bumping into #2602.

**Reply by vladak** [reply]
For the record: in my Docker instance, I can see the same problem.

**Reply by jkjha** [reply]
Thanks for the issue link and update. while waiting for fix or better workaround, I have added nginx configuration to block all request where url matches <.p4config>

```
        location ~* \.p4config {
         return 404 "Sorry, the requested resource was not found.";
    }
```
